### ISSUE TITLE
java.lang.ClassNotFoundException: double

### ISSUE THREAD

**Issue by TheMercuryBeat** [quarkusio/quarkus-quickstarts#999]
I have downloaded version 2.5.0.Final of reactive-messaging-websockets-quickstart and when I start it I get the following error:

```
Caused by: java.lang.ClassNotFoundException: double
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:455)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:405)
	at io.quarkus.smallrye.reactivemessaging.deployment.WiringHelper.toType(WiringHelper.java:229)
	at io.quarkus.smallrye.reactivemessaging.deployment.WiringProcessor.generateDocumentationItem(WiringProcessor.java:293)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:887)
```
However in version 2.4.2.Final it works fine.

**Comment by cescoffier** [comment]
The fix has already been applied in the main branch of Quarkus.

**Comment by TheMercuryBeat** [comment]
Oh, I'm sorry, I didn't know

Thanks!!

**Comment by cescoffier** [comment]
No problem!

**Comment by gsmet** [comment]
@TheMercuryBeat If I'm not mistaken, the fix is in 2.5.1.Final that has just been released.
